DQMOffline/Trigger/python/MssmHbbMonitor_cfi.py

```diff
@@ -12,8 +12,29 @@
     triggerobjbtag = cms.string("hltBTagCalo30x8CSVp0p92SingleWithMatching"),
     triggerSummary = cms.InputTag("hltTriggerSummaryAOD","","HLT"),
     triggerResults = cms.InputTag("TriggerResults","","HLT"),
-    offlineCSVPF = cms.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags"),
-    
+    offlineBtag = cms.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags"),
+    histoPSet = cms.PSet(
+       jetPt = cms.PSet (
+          nbins = cms.int32 (60),
+          xmin  = cms.double(0),
+          xmax  = cms.double(600),
+       ),
+       jetEta = cms.PSet (
+          nbins = cms.int32 (10),
+          xmin  = cms.double(-2.5),
+          xmax  = cms.double(2.5),
+       ),
+       jetPhi = cms.PSet (
+          nbins = cms.int32 (14),
+          xmin  = cms.double(-3.5),
+          xmax  = cms.double(3.5),
+       ),
+       jetBtag = cms.PSet (
+          nbins = cms.int32 (40),
```

Comment:
Will restrict the number of bins. Full distribution will be checked in the offline analysis as usual.